universal-login-commons/test/core/utils/validateEmail.test.ts

```diff
@@ -0,0 +1,16 @@
+import {validateEmail} from '../../../src/core/utils/validateEmail';
+import {expect} from 'chai';
+
+describe('UNIT: ValidateEmail', () => {
+  it('Should validate valid email', () => {
+    expect(validateEmail('valid@email.com')).be.eq(true);
+  });
```

Comment:
Wrige more tests. You can find more edgecases :
```
it('correct email',  () => {
 ...'
}

describe('invalid emal', () => {
  it('empty string', () => {
    validate('')..
}
'e@email'
'@email'
'somestring'
'.account@email.com'
'_account@email.com'
....
}
```

Reply:
For creating test u can use at least these rules: https://www.itsupportguides.com/knowledge-base/exchange-2010/email-address-rules/

Reply:
Done